Directory.Build.targets

```diff
@@ -5,10 +5,8 @@
 
   <!-- Source build properties -->
   <PropertyGroup Condition=" '$(DotNetBuildFromSource)' == 'true' ">
```

Comment:
I just realized the code I referred you to in Roslyn was not 100% correct.  I think this should be changed to the following:

```
<!--
  CI source build leg: this needs to build the current and previous source build TFM. Both are 
  necessary as the output of this leg is used in other CI source build legs. Those could be 
  targeting NetCurrent or NetPrevious hence we must produce both. 
 -->
<PropertyGroup Condition=" '$(DotNetBuildFromSource)' == 'true'  AND '$(DotNetBuildFromSourceFlavor)' != 'Product'">
  <TargetFrameworks>$(NetPrevious);$(NetCurrent)</TargetFrameworks>
</PropertyGroup>

<!-- 
  Source build the product: this is the all up build of the product which needs only NetCurrent
-->
<PropertyGroup Condition=" '$(DotNetBuildFromSource)' == 'true'  AND '$(DotNetBuildFromSourceFlavor)' == 'Product'">
  <TargetFrameworks>$(NetCurrent)</TargetFrameworks>
</PropertyGroup>
```